### ISSUE TITLE
Store last applied settings to be kept has the default settings

### ISSUE THREAD

**Issue by HoSiuFu** [HoSiuFu/enhanced-traffic-config#7]
## Description
Currently, the settings always fall onto the defaults provided by Beamng, although this is not a big deal it becomes tedious to have to configure everything again, especially if the user uses the same settings on all sessions.

There is also a fact that currently the form resets when something makes the UI reload, this probably is an issue with the way that it is being initialized.

The objective of this issue is to store the last applied parameters persistently so that the user can quickly apply the settings.
 
## More Context
To do this you can check the official Beamng.drive documentation [here](https://documentation.beamng.com/modding/ui/app_creation/)

**Comment by HoSiuFu** [comment]
Well, this issue will not be as easy as I predicted because, apparently, the example, that it's displayed on the documentation of Beamng, does not work. I've also tried to find something similar to the example and the only thing that I found was a Drag Race Test App that seems to be not working.

I will try a different approach by using Lua to make a base extension and use functions defined under Lua to save the settings persistently.
